backend/tests/services/kanban/Jira/JiraVerifyToken.test.ts

```diff
@@ -216,8 +196,187 @@ describe("verify token and return columns and users", async () => {
         flag: false,
       },
     ];
-    expectedResponse.users = jiraUserNames;
-    expectedResponse.jiraColumns = jiraColumnNames;
+    const expectedResponse = {
+      jiraColumns,
+      users,
+      targetFields,
+    };
+    const response = await jiraVerifyToken.verifyTokenAndGetColumnsAndUser(
+      tokenVerifyModel
+    );
+    expect(response.targetFields[0].flag).to.equal(false);
+    expect(response.users.length).to.equal(3);
     expect(response).deep.equal(expectedResponse);
   });
 });
+
+describe("get query status", () => {
+  it("should return query status data", async () => {
+    mock
+      .onGet("https://domain.atlassian.net/rest/api/2/status/10006", {
+        headers: { Authorization: "testToken" },
+      })
+      .reply(200, StatusData);
+    const result: StatusSelf =
+      await jiraVerifyTokenProto.constructor.queryStatus(
+        "https://domain.atlassian.net/rest/api/2/status/10006",
+        "testToken"
+      );
+    expect(result.statusCategory.key).equal("indeterminate");
+  });
+});
+
+describe("get all done cards", () => {
+  const storyPointsAndCycleTimeRequest = new StoryPointsAndCycleTimeRequest(
+    "testToken",
+    "jira",
+    "domain",
+    "project",
+    "2",
+    ["Done"],
+    1589080044000,
+    1589944044000,
+    [
+      {
+        key: "customfield_10016",
+        name: "Story point estimate",
+        flag: true,
+      },
+    ],
+    false
+  );
+
+  it("should return all done cards", async () => {
+    jiraVerifyTokenProto.httpClient = axios.create({
+      baseURL: "https://domain.atlassian.net/rest/agile/1.0/board",
+    });
+    jiraVerifyTokenProto.httpClient.defaults.headers.common["Authorization"] =
+      "testToken";
+    jiraVerifyTokenProto.queryCount = 100;
+    mock
+      .onGet(
+        `https://${
+          tokenVerifyModel.site
+        }.atlassian.net/rest/agile/1.0/board/2/issue?maxResults=100&jql=status in ('${doneColumn.join(
+          "','"
+        )}') AND statusCategoryChangedDate >= ${
+          tokenVerifyModel.startTime
+        } AND statusCategoryChangedDate <= ${tokenVerifyModel.endTime}`,
+        {
+          headers: { Authorization: `${tokenVerifyModel.token}` },
+        }
+      )
+      .reply(200, JiraCards);
+
+    const response = await jiraVerifyTokenProto.getAllDoneCards(
+      tokenVerifyModel,
+      doneColumn
+    );
+
+    expect(response.length).deep.equal(1);
+  });
+});
+
+describe("make page query", () => {
+  it("should return cards issues ", async () => {
```

Comment:
given

when

then

this is a good pattern, PLZ change the other test to match this pattern